packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -374,6 +375,63 @@ describe('Meter', () => {
         assert.strictEqual(record1.aggregator.toPoint().value, 20);
         assert.strictEqual(boundCounter, boundCounter1);
       });
+
+      it('should trunk non-integer values for INT valueType', async () => {
+        const upDownCounter = meter.createUpDownCounter('name', {
+          valueType: ValueType.INT,
+        });
+        const boundCounter = upDownCounter.bind(labels);
+        {
```

Comment:
The brackets give lexical scope to the `const [record1] = meter.getBatcher().checkPointSet();` line. It is repeated several times in the test so that different values may be tested.

These should be separate tests as they are not all testing truncation